setup.py

```diff
@@ -0,0 +1,24 @@
+"""
+Code With fseai
+"""
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="fseai",
+    version="0.0.1",
+    author="Hanoona Rasheed",
+    author_email="hanoona@uniqueroboticsedu.com",
+    description="Full Stack Engineering AI course package",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    packages=setuptools.find_packages(),
+    classifiers=[
+        "Programming Language :: Python :: 3",
```

Comment:
take a look at the classifiers, you could add more
- Development status
- Environment
- Operating system
- Programming language (You have added python 3 already, but additionally, it's good to mention the minor version too, for example : `Programming Language :: Python :: 3.6`. You can see the whole list of classifiers [here](https://pypi.org/classifiers/)